src/app.ts

```diff
@@ -86,6 +82,52 @@ export default (
       .inc();
   };
 
+  const createTaskRun = (
+    name: string,
+    context: any,
+    checkRunId: string = ''
+  ) => {
+    var params = [
+      {
+        name: 'SECRET_NAME',
+        value: getTokenSecretName(context),
+      },
+    ];
+    if (checkRunId !== '') {
+      params.push({
+        name: 'CHECK_RUN_ID',
+        value: checkRunId,
+      });
+    }
```

Comment:
Let's use this instead:

```suggestion
    extraParams: Array<Record<string, unknown>> = []
  ) => {
    const params = [
      {
        name: 'SECRET_NAME',
        value: getTokenSecretName(context),
      },
      ...extraParams
    ];
```

and later call this as:

```ts
   createTaskRun('peribolos-run', context, [{
     name: 'CHECK_RUN_ID',
     value: checkRunId,
   }])
```

Reply:
Resolved in a pushed commit.